$NetBSD: patch-bc,v 1.3 2015/12/08 11:13:04 jperkin Exp $

replacing explicit --export-dynamic by ${EXPORT_SYMBOLS_LDFLAGS}
for xlc

--- m4/libtool.m4.orig	2009-08-23 11:38:19.000000000 +0000
+++ m4/libtool.m4
@@ -5947,7 +5947,7 @@ if test "$_lt_caught_CXX_error" != yes;
 	  xl*)
 	    # IBM XL 8.0 on PPC, with GNU ld
 	    _LT_TAGVAR(hardcode_libdir_flag_spec, $1)='${wl}-rpath ${wl}$libdir'
-	    _LT_TAGVAR(export_dynamic_flag_spec, $1)='${wl}--export-dynamic'
+	    _LT_TAGVAR(export_dynamic_flag_spec, $1)='${EXPORT_SYMBOLS_LDFLAGS}'
 	    _LT_TAGVAR(archive_cmds, $1)='$CC -qmkshrobj $libobjs $deplibs $compiler_flags ${wl}-soname $wl$soname -o $lib'
 	    if test "x$supports_anon_versioning" = xyes; then
 	      _LT_TAGVAR(archive_expsym_cmds, $1)='echo "{ global:" > $output_objdir/$libname.ver~
